torch/csrc/jit/script/module.h

```diff
@@ -400,6 +422,16 @@ struct TORCH_API Module {
       const std::string& filename,
       const ExtraFilesMap& extra_files = ExtraFilesMap());
 
+  void save_method(
```

Comment:
This is user-facing public API, it cannot be modified in service of a test.

Reply:
I also realize that it's user-facing. "it cannot be modified in service of a test", does it mean that if we want to add a user-facing function, a test must be added? I've included test_lite_executor.h in this pull. 
The user-facing idea in my mind is that for PyTorch script, we provide the option for users to dump instruction-level byte codes. If they want to have inference on a method, they could dump instructions using this function. On device the light executor could load and execute them. All other functionalities in script::module are kept, such as the original save/load functions. Any other suggestions on how would a user use this feature?